### ISSUE TITLE
Doesn't advice clojurescript 0.0-2371 -> 1.7.170

### ISSUE THREAD

**Issue by edbond** [xsc/lein-ancient#69]
I have old project with
```
                 [org.clojure/clojurescript "0.0-2371"]
```
lein ancient tells "nothing to upgrade".

**Comment by xsc** [comment]
@edbond Could you add the version of lein-ancient you're using? Version 0.6.8 treats ClojureScript like Clojure, requiring the `:check-clojure` command line flag for it to be checked, so can you try:

```
DEBUG=1 lein ancient :check-clojure
```

**Comment by edbond** [comment]
Yes, the version is latest :check-clojure does help, thanks.
Why this option is not enabled by default?

```
Leiningen's classpath: /home/eduard/.lein/self-installs/leiningen-2.5.3-standalone.jar
WARNING: update already refers to: #'clojure.core/update in namespace: clj-http.client, being replaced by: #'clj-http.client/update
Applying task ancient to [:check-clojure]
(debug) repositories: {"central" {:url "https://repo1.maven.org/maven2/", :snapshots false}, "clojars" {:url "https://clojars.org/repo/"}}
(debug) cli options: {:exclude [], :dependencies? true, :only [], :plugins? false, :recursive? false, :interactive? false, :print? false, :tests? true, :profiles? true, :check-clojure? true, :qualified? false, :snapshots? false, :colors? true}
(debug) repositories: nil
(debug) -- artifact [org.clojure/clojure "1.7.0"] at [:dependencies 0] ... (2 repositories)
(debug) -- artifact [clj-yaml "0.4.0"] at [:dependencies 5] ... (2 repositories)
(debug) -- artifact [cljs-http "0.1.39"] at [:dependencies 3] ... (2 repositories)
(debug) -- artifact [reagent "0.6.0-alpha"] at [:dependencies 4] ... (2 repositories)
(debug) -- artifact [org.clojure/core.async "0.2.374"] at [:dependencies 2] ... (2 repositories)
(debug) -- artifact [org.clojure/clojurescript "0.0-2371"] at [:dependencies 1] ... (2 repositories)
(debug) [central] - Δ  533ms - no results for reagent/reagent.
(debug) [central] - Δ  533ms - no results for cljs-http/cljs-http.
(debug) [central] - Δ  560ms - 17 versions found for org.clojure/core.async.
(debug) [central] - Δ  562ms - 86 versions found for org.clojure/clojure.
(debug) [central] - Δ  566ms - 133 versions found for org.clojure/clojurescript.
(debug) [central] - Δ  597ms - no results for clj-yaml/clj-yaml.
(debug) [clojars] - Δ  901ms - 6 versions found for clj-yaml/clj-yaml.
(debug) [clojars] - Δ  903ms - 16 versions found for reagent/reagent.
(debug) [clojars] - Δ  913ms - 7 versions found for org.clojure/clojure.
(debug) [clojars] - Δ  915ms - no results for org.clojure/clojurescript.
(debug) [clojars] - Δ  915ms - no results for org.clojure/core.async.
(debug) [clojars] - Δ  920ms - 52 versions found for cljs-http/cljs-http.
(debug) -- artifact [org.clojure/clojurescript "0.0-2371"] is outdated. (latest: "1.7.189")
```

**Comment by xsc** [comment]
Well, IMO, at least upgrading Clojure(Script) should be a conscious effort but maybe it does not make sense to exclude it from the check output (maybe with a hint towards `:check-clojure`). Thoughts?

**Comment by edbond** [comment]
Yes, I think it may be listed in other section (with notice that such artifacts not auto-upgraded) or with different color? and not upgrade automatically by upgrade command.

**Comment by xsc** [comment]
@edbond Can you try the latest SNAPSHOT? It now always displays outdated Clojure/ClojureScript dependencies but will neither include them when calculating lein-ancient's exit code nor automatically upgrade them.

```clojure
[lein-ancient "0.6.9-SNAPSHOT"]
```

Example Output:

```
[peripheral "0.5.0"] is available but we use "0.4.7"
[prismatic/schema "1.1.0"] is available but we use "1.0.5"
[com.taoensso/encore "2.46.1"] is available but we use "2.37.0"
[org.clojure/clojurescript "1.8.34"] is available but we use "1.7.228" (use :check-clojure to upgrade)
[ch.qos.logback/logback-classic "1.1.6"] is available but we use "1.1.5"
```

**Comment by edbond** [comment]
Works for me, more helpful output, thanks!

```
[org.clojure/clojure "1.8.0"] is available but we use "1.7.0" (use :check-clojure to upgrade)
[org.clojure/clojurescript "1.8.34"] is available but we use "0.0-2371" (use :check-clojure to upgrade)
```

**Comment by xsc** [comment]
Released as:

```clojure
[lein-ancient "0.6.9"]
```

[&raquo; Release Notes](https://github.com/xsc/lein-ancient/releases/tag/clojars-0.6.9)
